imageroot/templates/reports-api.j2

```diff
@@ -0,0 +1,111 @@
+{
+    "listen_address": "0.0.0.0:{{ reports_api_port }}",
```

Comment:
Can we listen on 127.0.0.1?

Reply:
Maybe? :man_shrugging:
That's uncharted territory, needs testing